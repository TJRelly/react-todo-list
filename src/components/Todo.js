import Button from "react-bootstrap/Button";

const Todo = ({ text, remove, id }) => {
    const removeTodo = () => remove(id);
    return (
        <div className="m-2">
            {text}
            <Button className="btn-danger ms-2" onClick={removeTodo}>
                delete
            </Button>
        </div>
    );
};

export default Todo;
