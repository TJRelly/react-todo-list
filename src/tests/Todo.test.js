import { render } from "@testing-library/react";
import Todo from "../components/Todo";

test("renders learn react link", () => {
    render(<Todo />);
});

test("matches snapshot", function() {
    const {asFragment} = render(<Todo />);
    expect(asFragment()).toMatchSnapshot();
  });
  