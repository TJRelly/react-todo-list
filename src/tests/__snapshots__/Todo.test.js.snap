// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="m-2"
  >
    <button
      class="btn-danger ms-2 btn btn-primary"
      type="button"
    >
      delete
    </button>
  </div>
</DocumentFragment>
`;
